listenbrainz/webserver/templates/index/index.html

```diff
@@ -4,9 +4,10 @@
     <div class="alert alert-success center-p" role="alert">
       ListenBrainz Year in Music for 2021 just dropped! Find yours <a href="{{ url_for('redirect.redirect_year_in_music') }}" class="alert-link">here</a>. Share it across social media and compare your stats with your friends!
     </div>
-    <div class="col-md-7 col-lg-8">
-      <img src="{{ url_for('static', filename='img/logo_big.svg') }}" id="big-logo" />
-      <h2 class="page-title">Welcome to ListenBrainz!</h2>
+   <div class="col-lg-12">
+      <h2 class="page-title">
+        Share and Visualize your Musical Habits!
+      </h2>
       <p>
         ListenBrainz keeps tracks of what music you listen to and
         provides you with insights into your listening habits. We're
```

Comment:
It would be good to add line breaks for readability (shorter lines = easier to read), while we're in here improving the page.
I'd suggest these:
- one before "We're completely open-source"
- one before "We also have an API"
- one before "Once you have an account"

Reply:
Done!